src/plugins/discourse/mirror.test.js

```diff
@@ -339,13 +426,12 @@ describe("plugins/discourse/mirror", () => {
 
       const postsFromTopic = NullUtil.get(await fetcher.topicWithPosts(1))
         .posts;
-      expect(postsFromTopic.map(getId)).toEqual([p1]);
+      expect(postsFromTopic.map(getId)).toEqual([p1, p2, p3]);
```

Comment:
why do the mode 1 tests change?

Reply:
Because the MockFetcher changed. It's implementation details of `topicWithPosts` are now consistent with #1455 behavior change to the real fetcher. Along with the other changes to MockFetcher it probably warrants still splitting this PR.

Reply:
Resolved by moving to #1473